src/v2i-hub/CDASimAdapter/src/CDASimConnection.cpp

```diff
@@ -47,19 +53,20 @@ namespace CDASimAdapter{
         message["location"]["x"] = location.X;
         message["location"]["y"] = location.Y;
         message["location"]["z"] = location.Z;
+        message["sensors"] = sensors_json_v;
```

Comment:
You need to include the new simulatedInteractionPort in the handshake.

Reply:
It is in another PR.

Reply:
is there an order these two PRs need to be merged. Does this one need to be merged first?